src/modal/modal.ts

```diff
@@ -21,7 +21,7 @@ export interface NgbModalOptions {
   /**
    * Size of a new modal window.
    */
-  size?: 'sm' | 'lg';
+  size?: 'sm' | 'lg' | 'md';
```

Comment:
The size exists in the variables, but if I understand correctly, it isn't supposed to be used in the markup. The generated CSS is

```
@media (min-width: 576px) {
  .modal-dialog {
    max-width: 500px;
    margin: 30px auto;
  }
  .modal-sm {
    max-width: 300px;
  }
}

@media (min-width: 992px) {
  .modal-lg {
    max-width: 800px;
  }
}
```

IMHO the correct way to have the "normal" md size is to avoid specifying the size in the NgbModalOptions. Specifying an 'md' size would be equivalent in not specifying the size, in terms of the size of the rendered modal, but it would add a useless `.modal-md` CSS class to the markup.

I understand that it could be more convenient to specify an alternate size rather than not specifying it at all (or passing null or undefined), but adding a useless CSS class that isn't defined by bootstrap doesn't look right to me.

@pkozlowski-opensource what do you think?